engine/model/world.py

```diff
@@ -109,10 +110,10 @@ class World:
     objects_to_add: Dict[str, ScriptedObject]
 
     def __init__(
-        self,
-        api: scripts.GameAPI,
-        game_spec: spec.GameSpec,
-        initial_player_state: Dict[str, Any],
+            self,
+            api: scripts.GameAPI,
+            game_spec: spec.GameSpec,
+            initial_player_state: Dict[str, Any],
```

Comment:
What auto-formatter are you using? Since this is all code that you didn't write and I formatted it using black, it means the one you're using is not that one or isn't configured the same way.

You'll want to ensure your editor is using that same one so that it doesn't create a whole bunch of spurious changes like this.

Reply:
oops sorry. I reformat it